src/VCR_InterfaceTasks.cpp

```diff
@@ -72,43 +62,65 @@ bool init_read_adc1_task(const unsigned long& sysMicros, const HT_TASK::TaskInfo
     adc_1.setChannelScaleAndOffset(7, 1, 0);
     /* NOLINTEND */
 
-    hal_printf("Initialized ADC0 at %d (micros)\n", sysMicros); // NOLINT
-
+    hal_printf("Initialized ADC1 at %d (millis)\n", sys_time::hal_millis()); // NOLINT
     return true;
 }
 
-bool run_read_adc1_task(const unsigned long& sysMicros, const HT_TASK::TaskInfo& taskInfo)
+void run_read_adc1_task()
 {
 
     adc_1.sample(); // Samples all eight channels.
     adc_1.convert(); // Converts all eight channels.
-
-    return true;
+    hal_printf("ADC1 reading 0 %d\n", adc_1.data.conversions[0].raw); // NOLINT
 }
 
-HT_TASK::Task read_adc1_task = HT_TASK::Task(init_read_adc1_task, run_read_adc1_task, 10, 40000UL); // 20000us is 25Hz //NOLINT
-
-bool run_update_buzzer_controller_task(const unsigned long& sysMicros, const HT_TASK::TaskInfo& taskInfo)
+void run_update_buzzer_controller_task()
 {
+    vcr_data.system_data.buzzer_is_active = BuzzerController::getInstance().buzzer_is_active(sys_time::hal_millis()); //NOLINT
 
-    vcr_data.system_data.buzzer_is_active = BuzzerController::getInstance().buzzer_is_active(sysMicros / 1000); //NOLINT
-
-    return true;
 }
 
-HT_TASK::Task update_buzzer_controller_task = HT_TASK::Task(HT_TASK::DUMMY_FUNCTION, run_update_buzzer_controller_task, 10, 20000UL); // 20000us is 50hz //NOLINT
-
-
-bool create_watchdog(const unsigned long & sysMicros, const HT_TASK::TaskInfo &task_info)
+void create_watchdog()
 {
     WatchdogInstance::create(default_system_params::KICK_INTERVAL_MS); // this has issues for some reason with clang-tidy // NOLINT
-    return true;
 }
 
-bool run_kick_watchdog(const unsigned long& sysMicros, const HT_TASK::TaskInfo& taskInfo)
+void run_kick_watchdog()
 {
-    digitalWrite(WATCHDOG_PIN, WatchdogInstance::instance().get_watchdog_state(sysMicros / 1000));
+    digitalWrite(WATCHDOG_PIN, WatchdogInstance::instance().get_watchdog_state(sys_time::hal_millis()));
+}
+
+
+bool init_I2C(){
```

Comment:
nitpick: I'd rename this init_ioexpander_interface, since that's really what you're doing. This is the init function for interacting with a particular device.